CompfyuiOllama.py

```diff
@@ -16,7 +16,7 @@ async def get_models_endpoint(request):
     data = await request.json()
     url = data.get("url")
     client = Client(host=url)
-    models = [model['name'] for model in client.list().get('models', [])]
+    models = [model['model'] for model in client.list().get('models', [])]
```

Comment:
this should support both becuse on newer versions of ollama it works fine.

you can wrap it in try and catch and handle exception with calling model['model']